tests/components/websocket_api/test_commands.py

```diff
@@ -467,3 +468,46 @@ async def test_render_template_returns_with_match_all(
     assert msg["id"] == 5
     assert msg["type"] == const.TYPE_RESULT
     assert msg["success"]
+
+
+async def test_manifest_list(hass, websocket_client):
+    """Test loading manifests."""
+    http = await async_get_integration(hass, "http")
+    websocket_api = await async_get_integration(hass, "websocket_api")
+
+    await websocket_client.send_json({"id": 5, "type": "manifest/list"})
+
+    msg = await websocket_client.receive_json()
+    assert msg["id"] == 5
+    assert msg["type"] == const.TYPE_RESULT
+    assert msg["success"]
+    assert sorted(msg["result"], key=lambda manifest: manifest["domain"]) == [
+        http.manifest,
+        websocket_api.manifest,
+    ]
+
+
+async def test_manifest_get(hass, websocket_client):
+    """Test getting a manifest."""
+    hue = await async_get_integration(hass, "hue")
+
+    await websocket_client.send_json(
+        {"id": 6, "type": "manifest/get", "integration": "hue"}
+    )
+
+    msg = await websocket_client.receive_json()
+    assert msg["id"] == 6
+    assert msg["type"] == const.TYPE_RESULT
+    assert msg["success"]
+    assert msg["result"] == hue.manifest
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5365808)